cmd/podman/images/import.go

```diff
@@ -97,6 +101,22 @@ func importCon(cmd *cobra.Command, args []string) error {
 	default:
 		return errors.Errorf("too many arguments. Usage TARBALL [REFERENCE]")
 	}
+
+	if source == "-" {
+		outFile, err := ioutil.TempFile(util.Tmpdir(), "podman")
```

Comment:
I don't think you need the util.TmpDir() here, why not just pass ""?

Reply:
Can't you just set source to "/dev/stdin"?

Reply:
> Can't you just set source to "/dev/stdin"?

That's what we did before which caused the regression.  This code is directly copied from `load`; I think `util.TempDir()` is an extra security net for TMPDIR and falling back to /var/tmp.

Reply:
So ioutil.TempFile does not use $TMPDIR?

Reply:
https://pkg.go.dev/os#TempDir
Says that ioutil.TmpFile will end up using TMPDIR if set, but fall back to /tmp.
But we already force TMPDIR to always be set.

https://github.com/containers/podman/blob/main/cmd/podman/root.go#L178